app/helpers/dd20_format_validation.py

```diff
@@ -0,0 +1,56 @@
+import pandas as pd
+import hashlib
+
+
+def validate_dd20_format(data_frame: pd.DataFrame, expected_hash: str) -> bool:
+    """
+    compares header hash of dataframe wit hexpected hash value
+
+    Parameters
+    ----------
+    data_frame: pd.DataFrame
+        dd20 data frame station data or line data
+    expected_hash: str
+        the expected known hash value that must match the computed hash of the dd20 dataframe headers 
+
+    Returns
+    -------
+    bool
+        true if hash values match.
+
+    """
+
+    return calculate_dd20_format_hash(data_frame) == expected_hash
+
+
+def calculate_dd20_format_hash(data_frame: pd.DataFrame) -> str:
+    """
+    summarizes the hash value for each cell in the
+    first 3 rows (which are considered static header rows).
+    We do this to try to detect if the file format has changed.
+
+
+    Parameters
+    ----------
+    data_frame: pd.DataFrame
+        dd20 data frame station data or line data
+
+    Returns
+    -------
+    str
+        the hash value of the headers of the dd20 sheet
+
+    """
+
```

Comment:
```suggestion
    """Calculate hash value for first 3 rows in data frame

    Convert header-rows to a string and calculate a hash
    from them (they are considered static header rows).
    Used to detect if the file format has changed.

    Parameters
    ----------
    data_frame: pd.DataFrame
        dd20 data frame

    Returns
    -------
    str
        Calculated hash value of header rows
    """
```
I think the current description is a bit misleading compared to what it actually does.

Reply:
it should be row 1,2 not including the first row. So yes the description should be updated